pynicotine/gtkgui/ui/settings/userinterface.ui

```diff
@@ -59,8 +59,8 @@
                 </child>
                 <child>
                   <object class="GtkCheckButton" id="TabSelectPrevious">
-                    <property name="label" translatable="yes">Remember previous primary tab on startup</property>
-                    <property name="tooltip_text" translatable="yes">Start with Search Files by default.</property>
+                    <property name="label" translatable="yes">Restore the previously active Main Tab on start-up</property>
```

Comment:
Any reason for capitalizing main tab?
Isn't "startup" more common in the context of software?

Reply:
> Any reason for capitalizing main tab?
 I did that because it's the subject of the sentence. Maybe I'm a bit confused by the title case. :-}

> Isn't "startup" more common in the context of software?
Yes, you are right, in the context of computers it is.

Reply:
Then again, 'Search Files' in the next sentence uses Title Case. What is your reasoning behind the use of Title Case or not?

Reply:
To identify the label name of the main tab that will be started with, as opposed to some other action related to file searching in general.

Title Case is generally not used in the interface apart from the labels of the main tabs and other UI components such as the Log History pane for example.